cfme/tests/test_rest.py

```diff
@@ -1171,6 +1165,38 @@ def test_import_report(rest_api):
     assert response['message'] == 'Skipping Report (already in DB): [{}]'.format(menu_name)
 
 
+@pytest.mark.parametrize(
+    "from_detail", [True, False],
```

Comment:
So, one comment here, I would say that in this case it would be fine to split this into two tests and not use parametrize, but if you want to keep it as one test and parametrize, then maybe the asserts and reload should happen at the end and we push the from_detail into an array to just DRY things up a little. Honestly I'm fine with either splitting into single tests, or the above.